app/models/like.rb

```diff
@@ -1,2 +1,8 @@
 class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly use `after_save` to call `update_likes_counter` every time a new post has been saved to the database, then make the `update_likes_counter` method private. You can read more about `after_save` from [here](https://api.rubyonrails.org/v7.0.4/classes/ActiveRecord/Callbacks/ClassMethods.html#method-i-after_save:~:text=after_save(*args%2C%20%26block)).